src/main/java/com/iexec/core/replicate/ReplicatesService.java

```diff
@@ -403,8 +408,7 @@ public void updateReplicateStatus(String chainTaskId,
     public void updateReplicateStatus(OptimisticLockingFailureException exception,
```

Comment:
The `@Recover` (`void`) and `@Retryable` (`Optional`) methods don't have the same return type.
It seems weird.